spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -1,20 +1,49 @@
 # frozen_string_literal: true
 
+require 'rails_helper'
+
 RSpec.describe Api::V1::KeywordsController, type: :request do
   describe 'GET#index' do
-    it 'returns success status' do
-      get api_v1_keywords_path, headers: create_token_header
+    context 'given a logged in user gets theirs keyword' do
+      it 'returns success status' do
```

Comment:
I think we can improve the spec here a bit more structure like below

```suggestion
    context 'given a logged in' do
      context 'given the user query their own keyword' do
        it 'returns success status' do
          # ..
        end
      end

      context 'given the user has no keyword' do
        it 'returns success status' do
          # ..
        end
      end
    end
    
    context 'given non-logged in user' do
    
    end
```